src/components/Avatar.js

```diff
@@ -1,12 +1,76 @@
 import * as React from "react";
-import {Avatar as MUIAvatar} from "@mui/material";
+import { Avatar as MUIAvatar, Button, ClickAwayListener, Divider, Box, Popper, Typography } from '@mui/material';
+import { useRef, useState } from 'react';
+import "./Avatar.css";
+export default function Avatar({imageLink, logout, email, name}) {
+  const anchorRef = useRef(null);
+  const [open, setOpen] = useState(false);
+
+  const handleToggle = () => {
+    setOpen(!open);
+  };
+
+  const handleClose = (event) => {
+    if (anchorRef.current && anchorRef.current.contains(event.target)) {
+      return;
+    }
+    setOpen(false);
+  };
 
-export default function Avatar({imageLink}) {
   return (
-    <MUIAvatar
-      sx={{width: 48, height: 48}}
-      src={imageLink}
-      imgProps={{referrerpolicy: "no-referrer"}}
-    />
+    <>
+      <MUIAvatar
+        sx={{width: 48, height: 48}}
+        src={imageLink}
+        ref={anchorRef}
+        onClick={handleToggle}
+        imgProps={{referrerpolicy: "no-referrer"}}
+      />
+      <ClickAwayListener onClickAway={handleClose}>
+      <Popper open={open} anchorEl={anchorRef.current}>
+        <Box className="popper">
+        <Typography className={"name"}> {name}</Typography>
+          <Typography
+            sx={{
+              color: "rgba(0, 0, 0, 0.60)",
+              fontFamily: "Roboto",
```

Comment:
As discussed in the standup today, we should be having Nunito font across the website. There is some discrepancy in the figma file

Reply:
Done